src/routes/onboarding/add-to-homescreen.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import smRectangle from "../../images/Smallrectangle.svg";
+import shareBar from "../../images/Sharebar.svg";
+import lgRectangle from "../../images/Largerectangle.svg";
+import icons from "../../images/Addtohomescreenicons.svg";
+
+export const AddToHomescreen: React.FC = () => {
+  return (
+    <div className="flex flex-col bg-gradient-to-b from-[#F5F5F5] to-[#9ECCFA] overflow-auto max-w-screen-md pb-8 relative m-auto h-screen">

```

Comment:
The edges of the screen tightly hug the elements when the screen resolution drops, I'd suggest adding some padding (16px on each side according to the figma) so that the elements look a bit neater by never touching the edges of the screen. You can add 16px of padding to the left/right of an element using Tailwind's `px-4` class.